tests/wpt/web-platform-tests/fullscreen/model/remove-single.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<title>Remove the single element on the fullscreen element stack</title>
```

Comment:
Yeah, the sts was a typo that I edited in my original comment. Use tests/wpt/mozilla/tests/mozilla/. document-onfullscreenerror.html failing is expected, because it tests the error case.